caffeinated/admin/extend/events/Extend_Events_Admin_Page.core.php

```diff
@@ -1017,6 +1020,31 @@ public function active_status_dropdown($current_value = '')
         return EEH_Form_Fields::select_input($select_name, $values, $current_value, $id, $class);
     }
 
+    /**
+     * returns a list of "venues"
+     *
+     * @param  string $current_value whatever the current active status is
+     * @return string
+     */
+    protected function venuesDropdown($current_value = '')
+    {
+        $select_name = 'venue';
+        $values = array(
+            '' => esc_html__('All Venues', 'event_espresso'),
+        );
+        // populate the list of venues.
+        $vnumdl = EE_Registry::instance()->load_model('Venue');
```

Comment:
Please try to avoid using abbreviations for variable names, as there is zero benefit nowadays to having super short variable names like this. 
Readability is far more important, so this should be changed to something like `$venue_model`.

Abbreviations that are instantly recognizable because they are used extensively, such as `qty`, `min`, `max`, etc, are acceptable